content/en/security/application_security/setup/java/linux.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Setup App and API Protection for Java on Linux
+code_lang: linux
+type: multi-code-lang
+code_lang_weight: 30
+further_reading:
+- link: "/security/application_security/how-it-works/"
+  tag: "Documentation"
+  text: "How App and API Protection Works"
+- link: "/security/default_rules/?category=cat-application-security"
+  tag: "Documentation"
+  text: "OOTB App and API Protection Rules"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting App and API Protection"
+---
+
+{{< partial name="app_and_api_protection/callout.html" >}}
+
+{{< partial name="app_and_api_protection/java/overview.html" >}}
+
+This guide explains how to set up App and API Protection (AAP) for Java applications running on Linux. The setup involves:
+1. Installing the Datadog Agent
+2. Configuring your Java application
+3. Enabling AAP monitoring
+
+## Prerequisites
+
+- Linux operating system
+- Java application
+- Root or sudo privileges
+- Systemd (for service management)
+
+## Setup
+
+### 1. Install the Datadog Agent
+
+#### Using the one-line install script (recommended)
+
+```bash
+DD_AGENT_MAJOR_VERSION=7 DD_API_KEY=<YOUR_API_KEY> DD_SITE=<YOUR_DD_SITE> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+```
+
+#### Manual installation
+
+For Debian/Ubuntu:
+```bash
+sudo apt-get update
+sudo apt-get install -y apt-transport-https curl gnupg
+sudo sh -c "echo 'deb https://apt.datadoghq.com/ stable 7' > /etc/apt/sources.list.d/datadog.list"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A2923DFF56EDA6E76E55E492D3A80E30382E94DE
+sudo apt-get update
+sudo apt-get install -y datadog-agent
+```
+
+For RHEL/CentOS:
+```bash
+sudo yum install -y https://yum.datadoghq.com/rpm/x86_64/datadog-agent-7.x.x-1.x86_64.rpm
+```
+
+## Library setup
+
+To enable AAP capabilities, you need the Datadog Java tracing library (version 0.94.0 or higher) installed in your application environment.
```

Comment:
Please avoid saying AAP but rather App and API Protection full-text mode 🙏 (I know this flipped in many places already by that's what we agreed on with product)